### PR TITLE
Prototype Translations + Adding JustinBack and myself to src/about.html + Some SEO Improvements

### PR DESCRIPTION
**Pull request by DrKJeff16**
As contributors ourselves (and since I got the suggestion through Twitter: https://twitter.com/ToSDR/status/1314457814396526594), I added myself and also JustinBack to the about page. Of course, if any feedback or reject is given, will be open to discussion.

### CODE REVIEW

File: dist/about.html [hunk 1]
```diff
@@ -96,7 +96,7 @@ <h2 id="contact">Contact us <a href="http://twitter.com/tosdr" class="btn btn-bi
 				</p>
 				<h3>Requesting a service review</h3>
 				<p>
-				  Unfortunately, we do not have the contributor capacity to do on-demand reviews. However, volunteers can review terms of service using our tool at <a href="https://edit.tosdr.org">https://edit.tosdr.org</a>, so perhaps one of your readers would be willing to do that? You can also discuss it with other reviewers, at <a href="https://forum.tosdr.org">forum.tosdr.org</a>.
+				  Unfortunately, we do not have the contributor capacity to do on-demand reviews. However, volunteers can review terms of service using our tool at https://edit.tosdr.org, so perhaps one of your readers would be willing to do that? You can also discuss it with other reviewers, at https://forum.tosdr.org</a>.
```
Comment (DrKJeff16) [inline dist/about.html:99]: Deleted the <a> tags because they were over a link themselves lol.

Comment (JustinBack) [reply dist/about.html:99]: The tag has an enclosing </a> in the end

**Review by JustinBack — CHANGES_REQUESTED** [review]
about.html has an extra `</a>` in the end

**Review by JustinBack — CHANGES_REQUESTED** [review]
firefox_YHc2Dxw7ea

There are missing non breaking spaces which should fix the formatting quite a bit in L220

```html
<h3 id="justin">Justin Renè Back <small>Software Developer <a href="https://twitter.com/justinreneback">@justinreneback</a></small></h3>
```

grafik


Also I think for twitter links we should all use blank targets as well as for links to personal websites. Possibly even noref attributoes for SEO

**Comment by DrKJeff16** [comment]
> Also I think for twitter links we should all use blank targets as well as for links to personal websites. Possibly even noref attributoes for SEO

Thanks for the advice. Could you provide me a link to that for me to learn to do it? I also plan to add alt-text to the images as it's good for Search Results.

**Comment by JustinBack** [comment]
rel nofollow

https://www.w3schools.com/tags/att_a_rel.asp

target _blank

https://www.w3schools.com/tags/att_a_target.asp

**Comment by JustinBack** [comment]
I advise not to change anything in the bootstrap file but rather make an override in a new css file. Males debugging easier

**Comment by DrKJeff16** [comment]
> I advise not to change anything in the bootstrap file but rather make an override in a new css file. Makes debugging easier

Oh yeah, correcting that asap.

**Comment by michielbdejong** [comment]
Great!

Can you fix the conflicting files?

**Review by JustinBack — APPROVED** [review]

**Comment by JustinBack** [comment]
There are still conflicting files, you need to update your fork.

**Comment by DrKJeff16** [comment]
Fork updated :3

File: src/about.html [hunk 2]
```diff
@@ -61,7 +61,7 @@
 		</div>
 
 		<div class="hero-unit">
-			<section class="container">
+			<section class="container" style="z-index:2;background:url('img/tosdr-logo-128-low-opacity.png') no-repeat center;">
```
Comment (DrKJeff16) [inline src/about.html:64]: Background image. I could make it even more transparent if necessary.

File: dist/about.html [hunk 3]
```diff
@@ -67,158 +67,179 @@ <h1 class="page-header">About</h1>
...
+					(<a href="https://twitter.com/hugoroyd" target="_blank" rel="nofollow"><strong>@hugoroyd</strong></a>) started the legal analysis and brought the project to life. Since then, more people have joined the team and have contributed through the reviewer community. Ultimately, all the work is transparent and the discussions happen in public. Our work is <a href="thanks.html" target="_blank">funded</a> by non-profits organisations and individual donations and gets released as <a href="http://fsfe.org/about/basics/freesoftware.html">free software</a> and open data.</a>
 				</p>
 				<p>
-					Please bear in mind that the project is still in the early phase and that most data is subject to important changes. More information about <a href="classification.html">our classification</a>.
+					Please bear in mind that the project is still in the early phase and that most data is subject to important changes. More information about <a href="classification.html" target="_blank">our classification</a>.
```
Comment (JustinBack) [inline dist/about.html:74]:
```suggestion
					Please bear in mind that the project is still in the early phase and that most data is subject to important changes. More information about <a href="classification.html">our classification</a>.
```

There is no need for a target blank on internal sites

File: dist/about.html [hunk 4]
```diff
@@ -67,158 +67,179 @@ <h1 class="page-header">About</h1>
...
 				</p>
 				<p>
 					<strong>What does “ToS;DR” mean?</strong> the name is inspired by internet acronym TL;DR which stands for "Too Long; Didn't Read" and is often used on blogs and emails when a block of text is just really long and that people are too lazy to read the whole stuff. It was intended more as a code name than as a real name. But it seems that people like it (do you?)
 				</p>
 				<p>
-					<strong><a href="contribute.html" class="btn btn-big btn-primary">Contribute and help us!&nbsp;»</a></strong>&nbsp; You can help us in many ways: contribute energy, money, time or code :-)
+					<strong><a href="contribute.html" class="btn btn-big btn-primary" target="_blank">Contribute and help us!&nbsp;»</a></strong>&nbsp; You can help us in many ways: contribute energy, money, time or code :-)
```
Comment (JustinBack) [inline dist/about.html:80]:
```suggestion
					<strong><a href="contribute.html" class="btn btn-big btn-primary">Contribute and help us!&nbsp;»</a></strong>&nbsp; You can help us in many ways: contribute energy, money, time or code :-)
```

Same goes for this one, its internal, no need for blanks

File: dist/about.html [hunk 5]
```diff
@@ -39,21 +39,21 @@
 						<span class="icon-bar"></span>
 						<span class="icon-bar"></span>
 					</button>
-					<a class="brand" href="index.html"><img src="img/tosdr-icon-16.png" alt="" style="vertical-align:baseline;margin-right:.5em;" class="logo" />Terms of Service; Didn't&nbsp;Read</a>
+					<a class="brand" href="index.html"><img src="img/tosdr-icon-16.png" alt="Tos;DR Logo." style="vertical-align:baseline;margin-right:.5em;" class="logo" />Terms of Service; Didn't&nbsp;Read</a>
 					<div class="nav-collapse collapse">
 						<ul class="nav">
-							<li class=""><a href="index.html#services">Ratings</a></li>
+							<li class=""><a href="index.html#services" target="_blank">Ratings</a></li>
```
Comment (JustinBack) [inline dist/about.html:45]:
```suggestion
							<li class=""><a href="index.html#services">Ratings</a></li>
```

File: dist/about.html [hunk 6]
```diff
@@ -39,21 +39,21 @@
...
 <!--							<li class=""><a href="#classification">Classification</a></li>-->
 <!--							<li class=""><a href="#workinprogress">Progress</a></li>-->
-							<li class="active"><a href="about.html">About</a></li>
+							<li class="active"><a href="about.html" target="_blank">About</a></li>
```
Comment (JustinBack) [inline dist/about.html:48]:
```suggestion
							<li class="active"><a href="about.html">About</a></li>
```

File: dist/api/1/all.json [hunk 7]
```diff
@@ -1,6 +1,10 @@
 {
   "tosdr/api/version": 1,
+<<<<<<< HEAD
+  "tosdr/data/version": 1603998328337,
+=======
   "tosdr/data/version": 1603451310553,
```
Comment (JustinBack) [inline dist/api/1/all.json:6]:
```suggestion

```
Remove this as well, we've got the latest version above

File: dist/api/1/all.json [hunk 8]
```diff
@@ -1,6 +1,10 @@
 {
   "tosdr/api/version": 1,
+<<<<<<< HEAD
+  "tosdr/data/version": 1603998328337,
+=======
```
Comment (JustinBack) [inline dist/api/1/all.json:5]:
```suggestion

```

File: dist/api/1/all.json [hunk 9]
```diff
@@ -1,6 +1,10 @@
 {
   "tosdr/api/version": 1,
+<<<<<<< HEAD
+  "tosdr/data/version": 1603998328337,
```
Comment (JustinBack) [inline dist/api/1/all.json:4]: Keep this

File: dist/api/1/all.json [hunk 10]
```diff
@@ -1,6 +1,10 @@
...
+=======
   "tosdr/data/version": 1603451310553,
+>>>>>>> upstream/master
```
Comment (JustinBack) [inline dist/api/1/all.json:7]:
```suggestion

```

File: dist/api/1/all.json [hunk 11]
```diff
@@ -1,6 +1,10 @@
 {
   "tosdr/api/version": 1,
+<<<<<<< HEAD
```
Comment (JustinBack) [inline dist/api/1/all.json:3]: This breaks the JSON, remove github comments
```suggestion

```

File: dist/index.html [hunk 12]
```diff
@@ -1,179 +1,175 @@
...
+	<!-- Browser Donations: LedgerLoops -->
+	<link rel="ledger" href="wss://monetization-hubbie.herokuapp.com">
+	<!-- Browser Donations: Coil -->
+	<meta name="monetization" content="$twitter.xrptipbot.com/michielbdejong" />
+	<!--[if lt IE 9]><script src="https://html5shim.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
+	<!--Stylesheets.-->
+	<link href="css/bootstrap.css" rel="stylesheet">
+	<link href="css/bootstrap-responsive.css" rel="stylesheet">
+	<link href="css/custom.css" rel="stylesheet">
+	<link rel="shortcut icon" href="img/tosdr-icon-16.png">
```
Comment (JustinBack) [inline dist/index.html:28]:
```suggestion
	<link rel="shortcut icon" href="img/tosdr-icon-57.png">
```
Why not keep the higher res? This tag is favored by many crawlers and a higher resolution is always better

File: src/about.html [hunk 13]
```diff
@@ -39,186 +39,207 @@
 						<span class="icon-bar"></span>
 						<span class="icon-bar"></span>
 					</button>
-					<a class="brand" href="index.html"><img src="img/tosdr-icon-16.png" alt="" style="vertical-align:baseline;margin-right:.5em;" class="logo" />Terms of Service; Didn't&nbsp;Read</a>
+					<a class="brand" href="index.html"><img src="img/tosdr-icon-16.png" alt="Tos;DR Logo." style="vertical-align:baseline;margin-right:.5em;" class="logo" />Terms of Service; Didn't&nbsp;Read</a>
 					<div class="nav-collapse collapse">
 						<ul class="nav">
-							<li class=""><a href="index.html#services">Ratings</a></li>
+							<li class=""><a href="index.html#services" target="_blank">Ratings</a></li>
```
Comment (JustinBack) [inline src/about.html:46]:
```suggestion
							<li class=""><a href="index.html#services">Ratings</a></li>
```
No need for blanks on internal links

File: src/about.html [hunk 14]
```diff
@@ -39,186 +39,207 @@
...
 <!--							<li class=""><a href="#classification">Classification</a></li>-->
 <!--							<li class=""><a href="#workinprogress">Progress</a></li>-->
-							<li class="active"><a href="about.html">About</a></li>
+							<li class="active"><a href="about.html" target="_blank">About</a></li>
```
Comment (JustinBack) [inline src/about.html:49]:
```suggestion
							<li class="active"><a href="about.html">About</a></li>
```

File: src/about.html [hunk 15]
```diff
@@ -39,186 +39,207 @@
...
+							Donate <a href="https://opencollective.com/tosdr" target="_blank"><strong>On OpenCollective</strong></a>
 						</p>
 					</div>
 				</div>
 			</div>
 		</div>
 
 		<div class="hero-unit">
-			<section class="container">
+			<section class="container" style="z-index:2;background:url('img/tosdr-logo-128-low-opacity.png') no-repeat center;">
```
Comment (JustinBack) [inline src/about.html:66]: The image is not being copied over in the dist process, 404 in the dist folder

File: src/about.html [hunk 16]
```diff
@@ -39,186 +39,207 @@
...
 			<section id="contact">
 				<div class="page-header">
-					<h2 id="contact">Contact us <a href="http://twitter.com/tosdr" class="btn btn-big btn-info">Follow @tosdr on Twitter</a></h2>
+					<h2 id="contact">Contact us <a href="https://twitter.com/tosdr" target="_blank" rel="nofollow" class="btn btn-big btn-info">@tosdr</a> on Twitter</h2>
 				</div>
 				<style type="text/css">.zzz, .zzz a { float:right; font-size:.001px; color:transparent; display:inline-block; width:0px; text-decoration:none; }
 				</style>
 				<p>
-					You can reach us directly at <a href="https://forum.tosdr.org">our reviewers forum</a>. You can also contact us by email to:
+					You can reach us directly at <a href="https://forum.tosdr.org" target="_blank" style="display:block;text-align:center;text-decoration:none;color:#fff;background-image:linear-gradient(270deg,#159,#389,#37c);width:8.5em;border-radius:16px;padding:2px 8px;">Our reviewers forum</a>. You can also contact us by email at
```
Comment (JustinBack) [inline src/about.html:95]:
```suggestion
					You can reach us directly at <a href="https://forum.tosdr.org" target="_blank">Our reviewers forum</a>. You can also contact us by email at
```

That does not look really good and breaks the new lines. The link is already blue

grafik

Normal is wayyy better 

grafik

File: src/about.html [hunk 17]
```diff
@@ -219,120 +225,110 @@ <h3 id="Jeff">Guennadi Maximov Cortés <small>Self Taught <a href="https://twitt
...
+				<a href="https://github.com/xMartin">Martin Stadler</a>.
+			</p>
+		</section>
+		<section id="join-us" style="font-size:3em;text-align:center;">
+			<a href="contribute.html#join">Join Us</a>!
+		</section>
+	</article>
+	<footer id="bottom">
+		<div class="container">
+			<nav id="sitemap"> <!--TODO Replace Footer Sitemap with Bootstrap Columns.-->
```
Comment (DrKJeff16) [inline src/about.html:282]: Do you agree with this TODO?

**Review by JustinBack — CHANGES_REQUESTED** [review]
Quite a bit to improve, the blank targets should not be used on internal links as they open a plenty of new tabs. Also the link to the reviewers forum is waaaayy too unfitting and causes line breaks.

Could you wrap the `<section id="community">` inside of a `<div id="container">`

That fixes this 

grafik
and adds more spacing to the layout of the section

```html
<div id="container">
<section id="community">
</section>
</div>
```

File: dist/api/1/all.json (hunk 7 above)
Comment (JustinBack) [reply dist/api/1/all.json:6]: You can as well regenerate this with grunt then commit

**Comment by JustinBack** [comment]
I'm committing as much as I can to approve this PR

**Comment by DrKJeff16** [comment]
Standby, It got worse. Checking the latest commit.

**Comment by JustinBack** [comment]
Fixed this already ^^

File: dist/about.html [hunk 18]
```diff
@@ -44,10 +44,17 @@
 					<div class="nav-collapse collapse">
 						<ul class="nav">
 							<li class=""><a href="index.html#services">Ratings</a></li>
+<<<<<<< HEAD
```
Comment (DrKJeff16) [inline dist/about.html]: It's getting worse xD

Comment (JustinBack) [reply dist/about.html]: Nope thats a problem on your side, you need to recompile as you merged stuff rather than pulling the fork update

**Pull request merged**